Xcode/Testing-Swift/XMPPvCardTempTests.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  XMPPvCardTempTests.swift
+//  XMPPFrameworkSwiftTests
+//
+//  Created by Oleg Langer on 07.04.20.
+//
+
+import XCTest
+import XMPPFramework
+
```

Comment:
`import KissXML`

Reply:
Seems like I don't have to import anything at all.

Reply:
Ah yeah, it's probably implicitly re-exported by XMPPFramework.

Reply:
Oh, hm that's weird that you don't even need to import XMPPFramework.

Reply:
Yes, I also don't really understand why. :)